content/blog/choose-tidymodels-adventure/index.Rmd

```diff
@@ -0,0 +1,81 @@
+---
+output: hugodown::hugo_document
+slug: choose-tidymodels-adventure
+title: Choose your own tidymodels adventure
+date: 2021-05-24
+author: Julia Silge
+description: >
+    The tidymodels ecosystem is modular and flexible, which can sometimes make
+    choosing an approach overwhelming for newcomers. This post offers opinionated 
+    guidance on where to start!
+
+photo:
+  url: https://unsplash.com/photos/0LXFLzDOfuA
+  author: Jonathan Fox
+
+# one of: "deep-dive", "learn", "package", "programming", or "other"
+categories: [learn] 
+tags: [tidymodels, workflowsets, workflows, parsnip]
+---
+
+The [tidymodels](https://www.tidymodels.org/) framework is a collection of R packages for modeling and machine learning using tidyverse principles. You can install the tidymodels suite of packages from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("tidymodels")
+```
+
+The packages in tidymodels are designed to work together in a unified ecosystem, but they are flexible and modular; you can use tidymodels packages for certain parts of a modeling analysis without committing to it entirely, when appropriate.
+
+```{r setup}
+library(tidymodels)
+```
+
+We purposefully write code in small, modular packages both to make them easier to maintain and easier to use in production systems. This does mean that it can be challenging for a newcomer to know where their specific problem fits into this ecosystem. We generally recommend a couple of resources for folks just getting started:
+
+- the [**Get Started** section of tidymodels.org](https://www.tidymodels.org/start/) to get going quickly
+
+- our book [*Tidy Modeling with R*](https://www.tmwr.org/) to dig deeper
+
+This post takes on a more specific task for a newcomer to tidymodels; we consider several types of real-world modeling analyses and recommend ways to [choose your own adventure](https://en.wikipedia.org/wiki/Choose_Your_Own_Adventure) in the tidymodels ecosystem.
```

Comment:
Having read through the whole post and then coming back to this -- I would suggest saying here that the post will outline three scenarios and give a sentence description of each so (1) reader gets a sense of what they're in for and (2) someone who is only interested in one or two of them knows where to jump to. Perhaps most importantly, though, it's helpful to know that this is how the tidymodels team does a high-level categorization of "things" you can do with tidymodels: single model, workflow, workdlowset.